### PR TITLE
Add new package MED library (3.3.1)

### PR DESCRIPTION
**Pull request by 3rav**

### CODE REVIEW

File: mingw-w64-med/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Contributor: Rafa� Brzegowy <3rav@hotmail.com>
+
+_realname=med
+pkgname=mingw-w64-x86_64-${_realname}
+pkgver=3.3.1
+pkgrel=1
+pkgdesc="MED is the Data Echange Module based on the HDF library."
```
Comment (Alexpux) [inline mingw-w64-med/PKGBUILD]: Seems you miss ````x```` in ````Echange```` word. Remove dot at the end and add ````(mingw-w64)````

File: mingw-w64-med/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+arch=('any')
+url="https://www.salome-platform.org/downloads/current-version"
+license=('LGPL')
+depends=("${MINGW_PACKAGE_PREFIX}-hdf5")
+makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
+	     "${MINGW_PACKAGE_PREFIX}-cmake")
+options=('!strip' '!buildflags' 'staticlibs')
+source=("http://files.salome-platform.org/Salome/other/med-3.3.1.tar.gz")
```
Comment (Alexpux) [inline mingw-w64-med/PKGBUILD]: Dont hardcode version in URL. Use ````pkgver```` variable

**Comment by Alexpux** [comment]
@3rav package not buildable. It fail to link

**Comment by 3rav** [comment]
Hi,

I tested this library with gcc version 7.3 (the compilation ran correctly on my computer):

[bash output.txt](https://github.com/Alexpux/MINGW-packages/files/2018298/bash.output.txt)

**Pull request merged**